analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -723,29 +728,61 @@ def add_arguments_to_parser(parser):
         func=main, func_process_config_file=cmd_config.process_config_file)
 
 
-def __get_skip_handler(args) -> skiplist_handler.SkipListHandler:
+def get_source_file_paths(
+    file_filters: List[str],
+    compile_commands: tu_collector.CompilationDB,
+    header_file_extensions=(
+        '.h', '.hh', '.H', '.hp', '.hxx', '.hpp', '.HPP', '.h++', '.tcc')
```

Comment:
This shouldn't be a parameter of this function.

Reply:
Okay, I moved it to a separate variable outside of this function.